libs/api.js

```diff
@@ -1,5 +1,16 @@
 const mkAPI = client => {
   const api = {
+    createDoctype: async doctype => {
+      try {
+        return await client.fetchJSON('PUT', `/data/${doctype}/`)
+      } catch (e) {
+        if (e.reason && e.reason.error === 'file_exists') {
+          return { ok: true }
+        }
+        throw e
+      }
+    },
```

Comment:
A small test would be good ;) LGTM

Reply:
:gift: https://github.com/cozy/ACH/pull/40/commits/b7ff8b1c9448dc42ed53700602f358952701c506 :gift: